zipline/pipeline/factors/technical.py

```diff
@@ -101,6 +102,34 @@ def compute(self, today, assets, out, data):
         out[:] = nanmean(data, axis=0)
 
 
+class LinearWeightedMovingAverage(CustomFactor, SingleInputMixin):
+    """
+    Weighted Average Value of an arbitrary column
+
+    **Default Inputs**: None
+
+    **Default Window Length**: None
+    """
+    # numpy's nan functions throw warnings when passed an array containing only
+    # nans, but they still returns the desired value (nan), so we ignore the
+    # warning.
+    ctx = ignore_nanwarnings()
+
+    def compute(self, today, assets, out, data):
+        num_days = data.shape[0]
+
+        # Initialize weights array
+        weights = np.arange(1, num_days + 1, dtype=float)
+
+        # Compute normalizer
+        normalizer = (num_days * (num_days + 1)) / 2
+
+        for i in range(data.shape[1]):
```

Comment:
I'd be happy to make the suggested changes.  What's the best way to implement changes and get them pushed?